bin/tessel-2.js

```diff
@@ -197,6 +197,36 @@ parser.command('rename')
   .help('Change the name of a Tessel to something new.');
 
 
+// accessing the root shell of your tessels
+// Fixes issue https://github.com/tessel/t2-cli/issues/80
+/**
+$ t2 root --help
+> Usage: tessel root [-i <path>] [--help]
+>  -i <path>:   provide a path to the desired ssh key
+$ 
+
+$ t2 root
+> Accessing root...
+root@192.168.128.124 # 
+*/
+var default_id_rsa = '~/.tessel/id_rsa';
+var functional_msg = '\nGain SSH root access to one of your authorized tessels (menu listing if multiple targets)';
+parser.command('root')
+  .usage(functional_msg + '\n\nUsage: t2 root [-i PATH] [--help]\n\n-i PATH:   Optional targeting a different Private Key \n\n(Note: default target created by "t2 key generate" is ' + default_id_rsa + ')\n')
+  .option('path', {
+    abbr: 'i',
+    full: 'path',
+    metavar: 'PATH',
+    default: default_id_rsa,
```

Comment:
@johnnyman727 it is:
- [Tessel.TESSEL_AUTH_PATH + '/id_rsa'](https://github.com/Student007/t2-cli/blob/master/lib/lan_connection.js#L32) -> used in lan_connection.js
- [path.join(os.tmpDir(),'id_rsa')](https://github.com/ericvicenti/ssh-keygen/blob/master/src/ssh-keygen.js#L98) -> extern ssh-keygen module
- [filepath](https://github.com/Student007/t2-cli/blob/master/lib/tessel/provision.js#L3-L13) -> provision.js

`Tessel` is included in `provision.js` and also `lan_connection.js` but useless in `bin/tessel-2.js`!
The only solution would be to use `controller.js` to provide that *const* because `controller.js` is included within `bin/tessel-2.js`. A possible solution could be:
```
controller.TESSEL_AUTH_KEY = Tessel.TESSEL_AUTH_PATH+'/id_rsa';
```
within [controller.js](https://github.com/Student007/t2-cli/blob/master/lib/controller.js#L10)

That way I would replace `default: default_id_rsa` by `controller.TESSEL_AUTH_KEY`